imports/ui/Statistics.jsx

```diff
@@ -12,17 +20,46 @@ export default class Statistics extends Component{
     this.state={
       howManyEachClass: [],
       howManyReviewsEachClass: [],
-      totalReviews: -1
+      totalReviews: -1,
+      chartData: []
     }
     this.howManyEachClass();
     this.howManyReviewsEachClass();
     this.totalReviews();
+    this.getChartData();
+  }
+
+  getChartData(){
+    data=[];
```

Comment:
Declaring a variable without `var`, `let` or `const` will create a *global variable*!
You should never create global variable.
You should use `let` or `const`.